tests/auto/runtests.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+BASEDIR=$(dirname $0)
+PWD=`(cd $BASEDIR && pwd)`
+
+export LD_LIBRARY_PATH=$PWD/../../../qtmozembed/objdir-mer-dbg/src
```

Comment:
Ah.. you probably meant the script should be able to run in SDK. I'll need to install it first.